include/IndexStoreDB/Index/StoreUnitInfo.h

```diff
@@ -23,13 +23,13 @@ namespace index {
 struct StoreUnitInfo {
   std::string UnitName;
   CanonicalFilePath MainFilePath;
-  CanonicalFilePath OutFilePath;
+  StringRef OutFilePath;
```

Comment:
Might be nice to rename this to something like `OutFile` or `OutFileIdentifier`, but I'm also fine with it staying the way it is.